src/main/resources/static/assets/css/style-add-doctor.css

```diff
@@ -0,0 +1,2 @@
olor:rgba(237,85,100,.5);font-size:inherit;padding:0}.page-500{background:url(../content/500-page.jpg) 50% 50% no-repeat}.page-500 .content-box{background:hsla(0,0%,100%,.9)!important}.page-500 .content-box .content-body .icon{color:rgba(237,85,100,.5);font-size:inherit;padding:0}.accent-palette{color:#336cfb}.text-danger{color:#ed5564!important}.github-color{color:#24292e}.twitter-color{color:#1da1f2}.linkedin-color{color:#0073b1}.youtube-color{color:red}.nowrap{white-space:nowrap}.hidden{display:none!important}.no-style{background:none;border:0;outline:none;padding:0}.elements-list{align-items:flex-start;display:flex;flex-wrap:wrap;margin:-.35714rem}.elements-list>*{margin:.35714rem}.chat-container{display:flex;height:200px!important;position:relative;overflow:hidden}.chat-container.container-h-100{height:100px!important}.chat-container.container-h-50{height:50px!important}.chat-container.container-h-300{height:300px!important}@media (max-width:767px){.chat-container.container-h-300{height:200px!important}}.chat-container.container-h-400{height:400px!important}@media (max-width:767px){.chat-container.container-h-400{height:300px!important}}.chat-container.negative-indents{margin-top:-1.42857rem;margin-bottom:-1.42857rem}.rotate{-webkit-animation:spin 2s linear infinite;animation:spin 2s linear infinite}@-webkit-keyframes spin{to{transform:rotate(1turn)}}@keyframes spin{to{transform:rotate(1turn)}}.animated.delay-01s{-webkit-animation-delay:.1s;animation-delay:.1s}.animated.delay-02s{-webkit-animation-delay:.2s;animation-delay:.2s}.animated.delay-03s{-webkit-animation-delay:.3s;animation-delay:.3s}.animated.delay-04s{-webkit-animation-delay:.4s;animation-delay:.4s}.animated.delay-05s{-webkit-animation-delay:.5s;animation-delay:.5s}.animated.delay-06s{-webkit-animation-delay:.6s;animation-delay:.6s}.animated.delay-07s{-webkit-animation-delay:.7s;animation-delay:.7s}.animated.delay-08s{-webkit-animation-delay:.8s;animation-delay:.8s}.animated.delay-09s{-webkit-animation-delay:.9s;animation-delay:.9s}.rounded-20{border-radius:20px!important}.rounded-500{border-radius:500px!important}.btn-close{background:none;box-shadow:none!important;font-size:20px;font-weight:500;line-height:1;opacity:1;outline:none!important;text-shadow:none;transition:color .2s ease-in-out}.btn-close:hover{color:#ed5564}.map{height:100%}.icons-list{display:flex;flex-wrap:wrap;margin:-.5rem}.icons-list .icon{font-size:30px;margin:.5rem}.add-action-box{bottom:5.71429rem;position:fixed;right:1.42857rem;z-index:9997}.add-action-box .btn{font-size:30px;min-width:5rem!important}.department .team{flex-wrap:wrap}.department .team .team-img{box-shadow:0 18px 24px rgba(0,0,0,.12);margin-right:-10px}.bg-gradient{background:linear-gradient(90deg,#fff,#ebebeb)!important}.address-col{min-width:150px}.fs-20{font-size:20px}.fs-30{font-size:30px}.fs-48{font-size:48px}.personal-info-card .card-body{padding-top:0!important}.personal-info-card .user-actions{margin-top:-50px}.opacity-50{opacity:.5}
```

Comment:
please add a new css file and override there only the id or class that you want to change